pcdsdevices/epics_motor.py

```diff
@@ -56,15 +56,58 @@ class EpicsMotorInterface(FltMvInterface, EpicsMotor):
     disabled = Cpt(EpicsSignal, ".DISP", kind='omitted')
     # Description is valuable
     description = Cpt(EpicsSignal, '.DESC', kind='normal')
+    # Current Dial position
+    dial_position = Cpt(EpicsSignalRO, '.DRBV', kind='normal')
```

Comment:
I think this is the `dial` position that scientists wanted, if anybody could confirm - that would be great.

Reply:
Yes, that is what the scientists would call dial positions.

Reply:
great, thank you @silkenelson